.github/workflows/branch-deploy.yml

```diff
@@ -0,0 +1,98 @@
+name: Branch Based Deploy
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Should this be here for branch deploys?

Reply:
Yep, this bit means pull requests into the main branch

Reply:
Ahh makes sense, I think I missed the indentation!